:sparkles: (backend) Allow to create an order when one already exists but in state refunding/refunded (#1015)
openfun/joanie: CHANGELOG.md

- BO: Set properly the `is_graded` property on target course creation
- BO: Allow to use nested TranslatableForms

### Fixed

> - Allow a user to create a new order when the previous order already
>   exists with the triplet owner-course-product or owner-enrollment-product

Reviewer: ```suggestion
- Allow a user to create a new order when the previous order already
  exists but is is in `refunding` or `refunded` state
```
